lib/plugins/aws/provider/awsProvider.js

```diff
@@ -243,16 +243,21 @@ class AwsProvider {
                 e.statusCode !== 403 && // Invalid credentials
                 ((e.providerError && e.providerError.retryable) || e.statusCode === 429)
               ) {
+                const nextTryNum = numTry + 1;
+                const exponentialBackOffTime = Math.round(
+                  5000 + nextTryNum * Math.random() * 10000
+                );
+
                 that.serverless.cli.log(
                   _.join(
                     [
-                      `Recoverable error occurred (${e.message}), sleeping for 5 seconds.`,
-                      `Try ${numTry + 1} of ${MAX_TRIES}`,
+                      `Recoverable error occurred (${e.message}), sleeping for ${exponentialBackOffTime / 1000} seconds.`,
```

Comment:
Maybe let's change it to `sleeping for few seconds`

Reply:
We could do that, but I would prefer to have the exact number when looking at the logs. Especially now that we would move away from 5 seconds to a variable amount.

If you do not have a strong opinion about it I would leave it as is.

Reply:
We will not have fixed seconds number.. it'll be e.g. 4.234s, so it's a bit noisy.

Why do you find this output valuable? Specifically to confirm that jitter functionality works?